modin/core/storage_formats/base/query_compiler.py

```diff
@@ -2143,14 +2153,17 @@ def get_column(df, key):
 
         return DataFrameDefault.register(get_column)(self, key=key)
 
-    def getitem_row_array(self, key):
+    def getitem_row_array(self, key: List[Hashable], numeric: bool = False):
```

Comment:
I'm not sure where `numeric=False` is used? It seems that the old behavior has always been equivalent to `numeric=True`, and the same for the new behavior of all occurrences of `getitem_row_array`. Also, why is the default set to `False` rather than `True` which just by looking at this PR seems more commonly used?

Reply:
Devin's branch initially used `getitem_row_labels_array().getitem_column_array()` to do indexing with `loc`. I introduced the `numeric` parameter to `getitem_row_array` so we'd use `numeric=True` instead of the new method `getitem_row_labels_array`. Ultimately we decided we should use the new `take_2d_labels` instead of either `getitem_row_labels_array` or `getitem_row_array(numeric=True)`. `getitem_row_array` should now be as before.